xmtp-inbox-ios/Loaders/MessageLoader.swift

```diff
@@ -14,29 +14,35 @@ class MessageLoader: ObservableObject {
 	var client: XMTP.Client
 	var conversation: DB.Conversation
 
+	let fetchLimit = 10
```

Comment:
I made this 10 for now just to keep things light on the client, we can always bump it later if we want.